bundles/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/internal/HomekitImpl.java

```diff
@@ -211,60 +227,75 @@ private HomekitRoot startBridge(HomekitServer homekitServer, HomekitAuthInfoImpl
         return bridge;
     }
 
-    private void startHomekitServer() throws IOException, InvalidAlgorithmParameterException {
-        logger.trace("start HomeKit bridge");
-        if (homekitServers.isEmpty()) {
-            try {
-                networkInterface = InetAddress
-                        .getByName(((settings.networkInterface != null) && (!settings.networkInterface.isEmpty()))
-                                ? settings.networkInterface
-                                : networkAddressService.getPrimaryIpv4HostAddress());
-            } catch (UnknownHostException e) {
-                logger.warn("cannot resolve the Pv4 address / hostname {}.",
-                        networkAddressService.getPrimaryIpv4HostAddress());
+    private void startHomekitServer(int instance) throws IOException, InvalidAlgorithmParameterException {
+        logger.trace("starting HomeKit bridge instance {}", instance + 1);
```

Comment:
It seems like instance is sometimes 0 based, as seen in this function, and in other functions, 1 based (ie pruneDummyAccessories). This seems likely that some poor soul will get it wrong (and by poor soul, I mean myself). I'm wondering if we should normalize as 0-based at the UI level (parsing the command), and be consistent?

Reply:
agree, even for me who used to count many things starting from 0 in the past :) would be a little strange to refer to "instance 0". i would vote to use "1" and keep it consistence in the communication to the user.